lib/setup/setupBackup.js

```diff
@@ -594,17 +594,13 @@ class BackupRestore {
     }
 
     validateBackupAfterCreation(name) {
-        return new Promise((resolve, reject) => {
-            try {
-                const backupJSON = require(tmpDir + '/backup/backup.json');
-                if (!backupJSON.objects || !backupJSON.objects.length) {
-                    console.log('Backup does not contain valid objects');
-                    this.processExit(26);
-                } // endIf
-                resolve();
-            } catch (e) {
-                reject(e);
-            } // endTryCatch
+        return new Promise(resolve => {

```

Comment:
I find it weird that you handle one error (invalid objects) directly inside the promise and the case of corrupt files in the main file. I would probably reject the promise with the correct error message in both cases and then print the message and call process.exit in the main file.

Reply:
Idea was to handle errors directly where I know the error code and other errors where I have to exit with code 1 in the setup and do not have to check for if (err === 'xyz') but basically you are right and it seems inconsistent. I will change this.

Reply:
I think, now I get what you mean. I've looked at the other method ;-)